src/test/java/duke/task/EventTest.java

```diff
@@ -0,0 +1,22 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
+    @Test
+    public void test1() {
```

Comment:
Perhaps a good idea to give a brief idea of the test in the method name of the test rather than "test1()" so that readers can understand the general idea of what the test does